channeld/channeld.c

```diff
@@ -2955,6 +2971,23 @@ static struct amount_sat check_balances(struct peer *peer,
 	return funding_amount;
 }
 
+static int find_channel_funding_input(struct wally_psbt *psbt,
+				      struct bitcoin_outpoint *funding)
+{
+	for (size_t i = 0; i < psbt->num_inputs; i++) {
+		struct wally_psbt_input *in = &psbt->inputs[i];
+
+		if (0 != memcmp(in->txhash, &funding->txid,
+			       sizeof(in->txhash)))
+			continue;
+
+		if (funding->n == in->index)
+			return i;
```

Comment:
wally_psbt_input_get_outpoint... but I've said that so many times I think we need a bitcoin/psbt.h:

`struct wally_psbt_input *wally_psbt_get_input_spending(const struct wally_psbt *psbt, const struct bitcoin_outpoint *out, size_t *in_num)` where in_num is set if non-NULL (and return is non-NULL).

Reply:
Added this to a new github issue

Reply:
Added this to an issue https://github.com/ElementsProject/lightning/issues/6385